identity/src/keypair.rs

```diff
@@ -264,6 +264,38 @@ impl Keypair {
     }
 }
 
+#[cfg(feature = "ecdsa")]
+impl From<crate::ecdsa::Keypair> for self::Keypair {
+    fn from(kp: crate::ecdsa::Keypair) -> Self {
+        #[allow(deprecated)]
+        crate::Keypair::Ecdsa(kp)
+    }
+}
```

Comment:
```suggestion
#[cfg(feature = "ecdsa")]
impl From<ecdsa::Keypair> for Keypair {
    fn from(kp: ecdsa::Keypair) -> Self {
        #[allow(deprecated)]
        Keypair::Ecdsa(kp)
    }
}
```

I believe we can make this a bit more concise, same applies to the others.

Reply:
fixed